Library/Homebrew/cask/lib/hbc/artifact/pkg.rb

```diff
@@ -52,8 +54,19 @@ def run_installer(pkg_description)
         ]
         args << "-verboseR" if Hbc.verbose
         args << "-allowUntrusted" if pkg_install_opts :allow_untrusted
+        if pkg_install_opts :choices
+          choices_file = choices_xml
```

Comment:
BTW, I'm interested why that does not removes the file though `Tempfile` does so with GC according to doc.